README.rst

```diff
@@ -62,13 +67,24 @@ To enable this plugin:
     Scrapy setting to
     ``"twisted.internet.asyncioreactor.AsyncioSelectorReactor"``.
 
+In both cases:
+
 -   Set `your Zyte API key
     <https://docs.zyte.com/zyte-api/usage/general.html#authorization>`_ as
     either the ``ZYTE_API_KEY`` Scrapy setting or as an environment variable of
     the same name.
```

Comment:
If we don't auto-configure SCRAPY_POET_PROVIDERS, I think we should mention it here (and also in the example below).

Reply:
It's mentioned, I just need to merge main

Reply:
right :)